oval_graph/oval_node.py

```diff
@@ -16,60 +16,69 @@ class OvalNode():
     '''
     The OvalNode object is one node of oval_graph.
 
-    Args:
+    Require args:
```

Comment:
It's require**d**. You **require** them, therefore they are **required** by you.
And a small inspiration regarding docstrings - you can consider adopting the [Google format](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google), which is somehow standardized.